frontend/www/js/omegaup/components/problem/CollectionDetails.vue

```diff
@@ -1,20 +1,63 @@
 <template>
   <div>
     <h1 class="card-title">{{ title }}</h1>
+    <div class="row">
+      <div class="col col-md-4">
+        <div>
+          <omegaup-problem-filter-tags
+            :tags="collectionTags"
+            :public-tags="publicTags"
+            :selected-tags="selectedTags"
+            @add-tag="addTag"
+            @check="(selectedTags) => check(selectedTags)"
+          ></omegaup-problem-filter-tags>
+        </div>
+      </div>
+    </div>
   </div>
 </template>
 
 <script lang="ts">
 import { Vue, Component, Prop } from 'vue-property-decorator';
+import problem_FilterTags from './FilterTags.vue';
 import T from '../../lang';
 import { types } from '../../api_types';
 
-@Component
+@Component({
+  components: {
+    'omegaup-problem-filter-tags': problem_FilterTags,
+  },
+})
 export default class CollectionDetails extends Vue {
   @Prop() data!: types.CollectionDetailsPayload;
 
   T = T;
   type = this.data.type;
+  tags: string[] = [];
```

Comment:
```suggestion
  tags: string[] = this.data.collection.map((element) => element.alias);
```

así no se necesita `collectionTags`!